packages/next-codemod/transforms/lib/async-request-api/next-async-dynamic-prop.ts

```diff
@@ -812,3 +819,30 @@ function findAllTypes(
 
   return types
 }
+
+function commentSpreadProps(
+  path: ASTPath<FunctionScope>,
+  propsIdentifierName: string,
+  j: API['jscodeshift']
+) {
+  const functionBody = findFunctionBody(path)
+  const functionBodyCollection = j(functionBody)
+  // Find all the usage of spreading properties of `props`
+  const jsxSpreadProperties = functionBodyCollection.find(
+    j.JSXSpreadAttribute,
+    { argument: { name: propsIdentifierName } }
+  )
+  const objSpreadProperties = functionBodyCollection.find(j.SpreadElement, {
+    argument: { name: propsIdentifierName },
+  })
+  const comment = ` Next.js Dynamic Async API Codemod: '${propsIdentifierName}' is spread as props. Any asynchronous properties of 'props' must be awaited when accessed. `
```

Comment:
Should this also be `propsIdentifierName`? 

```suggestion
  const comment = ` Next.js Dynamic Async API Codemod: '${propsIdentifierName}' is spread as props. Any asynchronous properties of '${propsIdentifierName}' must be awaited when accessed. `
```